exercises/change/change.js

```diff
@@ -0,0 +1,14 @@
+//
+// This is only a SKELETON file for the 'Allergies' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+export class Candidate {
```

Comment:
According to [the `.spec` file](https://github.com/exercism/javascript/blob/952c01149428f1f1d8a82f65a37b8c02c5c95e42/exercises/change/change.spec.js), it's actually `Change` that is exported, not `Candidate`.